mitmproxy/options.py

```diff
@@ -30,19 +30,19 @@ def __init__(
             app_port: int = APP_PORT,
             anticache: bool = False,
             anticomp: bool = False,
-            client_replay: Sequence[str] = (),
+            client_replay: Sequence[str] = [],
```

Comment:
Correct, defaults should never be mutable, because every call gets the same instance if the default is used, so mutating it means all later calls get that mutated version too.